packages/manager/src/features/Linodes/LinodeCreatev2/utilities.ts

```diff
@@ -280,12 +348,14 @@ export const defaultValuesMap: Record<LinodeCreateType, CreateLinodeRequest> = {
  * Unfortunately, we have to wrap `yupResolver` so that we can transform the payload
  * using `getLinodeCreatePayload` before validation happens.
  */
-export const resolver: Resolver<CreateLinodeRequest> = async (
+export const resolver: Resolver<LinodeCreateFormValues> = async (
   values,
   context,
   options
 ) => {
-  const transformedValues = getLinodeCreatePayload(values);
+  const transformedValues = getLinodeCreatePayload(
+    values
+  ) as LinodeCreateFormValues;
```

Comment:
Can we avoid casting by adding the `linode` value back in?

```suggestion
  const transformedValues: LinodeCreateFormValues = {
    ...getLinodeCreatePayload(values),
    linode: values.linode,
  };
```